README.md

```diff
@@ -110,31 +110,22 @@ The original developers of RAJA are:
   * Rich Hornung (hornung1@llnl.gov)
   * Jeff Keasler (keasler1@llnl.gov)
 
-Contributors include:
-
-  * David Beckingsale (beckingsale1@llnl.gov)
-  * Jason Burmark (burmark1@llnl.gov)
-  * Holger Jones (jones19@llnl.gov)
-  * Will Killian (killian4@llnl.gov)
-  * Adam Kunen (kunen1@llnl.gov)
-  * Olga Pearce (pearce8@llnl.gov)
-  * David Poliakoff (poliakoff1@llnl.gov)
-  * Tom Scogland (scogland1@llnl.gov)
+Please see the {RAJA Contributors Page](https://github.com/LLNL/RAJA/graphs/contributors), to see the full list of contributors to the project.
```

Comment:
Rather than maintain contributor list in multiple places, I just added a link to it here.